model/operation.cc

```diff
@@ -678,6 +678,18 @@ namespace minsky
       r+=" [in2]="+ to_string(ports[2]->value());
     return r;
   }
+  
+  string DataOp::description_() const
+  {
+	return m_description;  
+  }
+   
+  string DataOp::description_(const std::string& x)
+  {
+    bb.update(*this); // adjust icon bounding box - see ticket #704
+    m_description=x;
```

Comment:
These two statements are done in the wrong order

Reply:
Thanks for catching this.